scripts/build.ts

```diff
@@ -28,18 +28,19 @@ const options = {
     'npm:multiformats@11.0.2/cid',
     'npm:multiformats@11.0.2/bases/base58',
     'npm:multiformats@11.0.2',
-    'npm:@multiformats/blake2@1.0.13'
+    'npm:@multiformats/blake2@1.0.13',
+    'npm:@chelonia/crypto@1.0.1'
   ],
   format: 'esm',
   outdir: 'build',
   splitting: false
 }
 const result = await esbuild.build(options)
-if (result.errors.length) {
+if ((result.errors as esbuild.Message[]).length > 0) {
```

Comment:
Not sure why this cast is needed, since the type should be implied (OTOH, I believe you might need a cast on `options` in line 37 to add `as esbuild.BuildOptions`)